draft-ietf-rtcweb-jsep.xml

```diff
@@ -2089,9 +2089,16 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               direction:
               <list style="symbols">
 
-                <t>An "a=msid" line, as specified in
-                <xref target="I-D.ietf-mmusic-msid"></xref>, Section
-                2.</t>
+                <t>For each MediaStream that was associated with the
+                transceiver when it was created via addTrack or addTransceiver,
+                an "a=msid" line, as specified in
+                <xref target="I-D.ietf-mmusic-msid"></xref>, Section 2. If a
+                MediaStreamTrack is attached to the transceiver's RtpSender,
+                the "a=msid" lines MUST use that track's ID. If no
+                MediaStreamTrack is attached, a valid ID MUST be generated
+                instead. To avoid fingerprinting, it is RECOMMENDED to generate
+                a UUID <xref target="RFC4122"></xref> using the form in section
```

Comment:
Any valid ID is allowed, the UUID is just a recommendation to avoid fingerprinting. I copied this recommendation from mediacapture-streams.

But your suggestion seems good to me (and there's less duplication that way). Whatever you prefer.